src/Opdex.Platform.Application.Abstractions/Models/Tokens/TokenSnapshotDto.cs

```diff
@@ -5,11 +5,7 @@ namespace Opdex.Platform.Application.Abstractions.Models.Tokens
 {
     public class TokenSnapshotDto
```

Comment:
I think we should add the token address for clients to reference if necessary, what do you think?

Reply:
Not in the individual token snapshot but in the page result, sure. It doesn't seem that useful to me honestly, as if you're making a request you have to already have the token and market address. Although I see no harm in adding it. I'll amend the `TokenSnapshotsResponseModel` and create a `MarketTokenSnapshotsResponseModel` so we can add the market address in the market tokens response as well.